packages/vue3/src/components/Skeleton/Skeleton.vue

```diff
@@ -34,6 +35,17 @@ withDefaults(
   animation: blink 2s linear infinite;
 }
 
+.card {
```

Comment:
```suggestion
.card, 
media {
```
cuz media and card has the same style